src/interfaces.ts

```diff
@@ -120,10 +120,15 @@ export interface PushRequest {
 
 export interface CreateRequest {
   model: string
-  path?: string
-  modelfile?: string
-  quantize?: string
   stream?: boolean
+  quantize?: string
```

Comment:
How are we planning to surface the error in case there is a mismatch between client and server?

Reply:
We should make sure the error from the server is re-thrown by the SDKs also, to centralize the behaviour in one place. 

This PR should follow this behaviour in the current form.